frontend/src/design/components/layout/DefaultSidebar.js

```diff
@@ -12,133 +7,144 @@ import {
-                icon: <ShareOutlined size={15} />
-              },
-              {
-                title: 'Glossaries',
-                path: '/console/glossaries',
-                icon: <BsIcons.BsTag size={15} />
-              }
-            ]
-          },
-          {
-            title: 'Play',
-            items: [
-              {
-                title: 'Worksheets',
-                path: '/console/worksheets',
-                icon: <AiOutlineExperiment size={15} />
-              },
-              {
-                title: 'Notebooks',
-                path: '/console/notebooks',
-                icon: <SiJupyter size={15} />
-              },
-              {
-                title: 'ML Studio',
-                path: '/console/mlstudio',
-                icon: <FiCodesandbox size={15} />
-              },
-              {
-                title: 'Pipelines',
-                path: '/console/pipelines',
-                icon: <BsIcons.BsGear size={15} />
-              },
-              {
-                title: 'Dashboards',
-                path: '/console/dashboards',
-                icon: <MdShowChart size={15} />
-              }
-            ]
-          },
-          {
-            title: 'Admin',
-            items: [
-              {
-                title: 'Organizations',
-                path: '/console/organizations',
-                icon: <BiIcons.BiBuildings size={15} />
-              },
-              {
-                title: 'Environments',
-                path: '/console/environments',
-                icon: <BsIcons.BsCloud size={15} />
-              }
-            ]
-          }
-        ];
+export const DefaultSidebar = ({ openDrawer, onOpenDrawerChange }) => {
+  const getSections = (isAdvancedMode) => {
+    const catalogSection = {
+      title: 'Catalog',
+      path: '/console/catalog',
+      icon: <VscBook size={15} />
+    };
+
+    const datasetsSection = {
+      title: 'Datasets',
+      path: '/console/datasets',
+      icon: <FiPackage size={15} />
+    };
+
+    const sharesSection = {
+      title: 'Shares',
+      path: '/console/shares',
+      icon: <ShareOutlined size={15} />
+    };
+
+    const glossariesSection = {
+      title: 'Glossaries',
+      path: '/console/glossaries',
+      icon: <BsIcons.BsTag size={15} />
+    };
+
+    const worksheetsSection = {
+      title: 'Worksheets',
+      path: '/console/worksheets',
+      icon: <AiOutlineExperiment size={15} />
+    };
+
+    const mlStudioSection = {
+      title: 'ML Studio',
+      path: '/console/mlstudio',
+      icon: <FiCodesandbox size={15} />
+    };
+
+    const dashboardsSection = {
+      title: 'Dashboards',
+      path: '/console/dashboards',
+      icon: <MdShowChart size={15} />
+    };
+
+    const notebooksSection = {
+      title: 'Notebooks',
+      path: '/console/notebooks',
+      icon: <SiJupyter size={15} />,
+      active: config.modules.notebooks.active
+    };
```

Comment:
3. This 'active' flag enables/disable the module from the side pane. See line 139 where we do the filter to show only active modules. In the future we will need to add this parameter to all modules